native/cocos/platform/ios/IOSPlatform.mm

```diff
@@ -150,6 +160,25 @@ - (void)renderScene:(id)sender {
     cc::events::WindowEvent::broadcast(ev);
 }
 
+void IOSPlatform::requestQuit() {
+    _requestQuit = true;
+    onClose();
+}
+
+void IOSPlatform::onDestroy() {
+    if(!_requestQuit) {
+        // ios exit process is special because it needs to wait for ts layer to destroy resources.
+        // The timer cannot be used here.
+        int32_t fps = getFps();
+        float sleepTime = 1000.0 / fps;
+        while (!_quitLoop) {
+            runTask();
+            usleep(sleepTime);
```

Comment:
While in `exit` progress,  is `sleep` needed here?  Should it exit ASAP ?

Reply:
If there will be no rendering involved, you should be able to remove the

Reply:
I think can remove it if not meet issues.

Reply:
OK